tests/unit/api.py

```diff
@@ -1222,6 +1252,58 @@ def test_list_projects(self, db):
             'runway',
         ]
 
+    def test_list_networks(self, db):
+        assert json.loads(api.list_networks()) == {}
+        api.project_create('anvil-nextgen')
+        network_create_simple('netA', 'anvil-nextgen')
+        result = json.loads(api.list_networks())
+        temp1 = uuid.UUID(result['netA']['driver_id'])
+        del result['netA']['driver_id']
+        assert result == {
+            'netA': {'projects': ['anvil-nextgen']}
+        }
+        api.network_delete('netA')
+        api.network_create('spiderwebs',
+                           creator='admin',
+                           access='',
+                           net_id='451')
+
+        result = json.loads(api.list_networks())
+        assert result == {
+            'spiderwebs': {'driver_id': '451', 'projects':None}
+        }
+
+    def test_list_network_attachments(self, db):
+        api.node_register('node-99', 'ipmihost', 'root', 'tapeworm')
```

Comment:
it was a change to the node_register call that was required by the OBM IPMI change - I didnt have it locally so I didnt run into the issue there and all the other tests were getting fixed by git when it merged. 

I just changed my calls to node_register to match the new syntax and it was fine